templates/web/panels/panel.tt.html

```diff
@@ -16,24 +12,17 @@ <h3 class="panel_title_[% panel.type %]">[% panel.title_element.title %]</h3>
         [% END %]
     [% END %]
 [% ELSE %]
-<ul data-accordion class="panel_accordion accordion" id="panel_[% panel_id | replace(':','-') %]"
-    [% IF panel.size == "small" %]
-        style="margin-top:0.2rem;margin-bottom:0.2rem;"
-    [% ELSE %]
-        style="margin-top:0.5rem;margin-bottom:1.5rem;"
-    [% END %]
->
+<ul data-accordion class="panel_accordion accordion" panel_margin_[% IF panel.size == 'small' %]small[% ELSE %]large[% END %]"
```

Comment:
The margin is not applied, e.g. those 2 panels are touching each other:
![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/1d66783a-4ade-4fc3-8815-dbc6912e2fbc)


```suggestion
<ul data-accordion class="panel_accordion accordion panel_margin_[% IF panel.size == 'small' %]small[% ELSE %]large[% END %]"
```